SparkPost/2web2ui
# FE-1166: Fix date selector for raw metrics with Report Builder

### What Changed
 - Simplified a lot of stuff.
 - Raw metrics always shows your local timezone.
 - Raw metrics always uses the recommended precision

### How To Test
- Without metrics rollup on both Hibana and OG:
     - Select 7 day range and submit. Then select 24 hours. The precision should be 24 hours.
     - In the date picker itself, select a time range < 24 hours. The text boxes for time should be editable because precision is < day. Then change to ~3 day time range. The text boxes should not be editable because time range = day.
     - Verify that the timezone is always your local timezone when changing date ranges.
- With metrics rollup on both Hibana and OG:
     - Select 7 day range and submit. Then select 24 hours. The precision should be days. Then select hour. The precision should change to 5 minutes.
     - In the date picker itself, select a time range < 24 hours. The text boxes for time should be editable because precision is < day. Then change to ~10 day time range. The text boxes should not be editable because time range = day.
     - Verify that changing precision and timezone still works.
 

### To Do
- [ ] There are still some wonky stuff. Discuss with UX. One solution would to always show recommended precision even with rollup.
